### ISSUE TITLE
CSS media hover not working in Firefox + back transition not working in Chrome

### ISSUE THREAD

**Issue by sedubois** [greglobinski/gatsby-starter-hero-blog#27]
- On Firefox, the mouse hover doesn't do anything when hovering above a blog post preview on the home page. On Chrome it works (the image gets zoomed and an orange arrow appears).
- On Chrome, the image gets zoomed in smoothly when hovering with the mouse, but it pops back without transition when taking the mouse out (I guess that a smooth transition would be better in this case too)

For the first point it seems fixed by removing `@media (hover: hover)` in `components/Blog/Item`, was that needed for something? And for the second point I have no idea as I'm a CSS newbie...

**Comment by greglobinski** [comment]
@sedubois 

First. Yes, Firefox doesn't supoprt `@media (hover: hover)` https://caniuse.com/#feat=css-media-interaction. These media queries prevent  adding hover effects on touch devices. 

Second. There is the code
```
 @media (hover: hover) {
            li {
              &:hover {
                :global(.gatsby-image-wrapper) {
                  transform: scale(1.1);
                  transition: all ${theme.time.duration.default};
                }
```
that means ` transition: all ${theme.time.duration.default};` is only applicable when the element is hovered, if it is not there is no the `transition` rule. You have to create the rule for a 'base' state of elements.

**Comment by sedubois** [comment]
OK, well hopefully Firefox will land support sooner or later (they're working on it). And for the transition I proposed the fix #28.
